packages/syft/src/syft/core/tensor/smpc/share_tensor.py

```diff
@@ -561,33 +635,36 @@ def __eq__(self, other: Any) -> bool:
         Returns:
             bool: True if equal False if not.
         """
+        # TODO: Rasswanth: Fix later after the comparison operation
         # relative
-        from .... import Tensor
+        # from .... import Tensor
 
-        if (
-            isinstance(self.child, Tensor)
-            and isinstance(other.child, Tensor)
-            and (self.child != other.child).child.any()  # type: ignore
-        ):
-            return False
+        # if (
```

Comment:
Q: Should those be commented?

Reply:
That we use it as proxy george, the singleentity phitensor passes down `eq` we do not really want to process it ,we should change afterwards on the implementation